lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -979,8 +983,34 @@ def restart_all_services(self):
             )
         elif self._init_system == InitMode.DOCKER:
             self.exec_command("cd /home/vagrant/magma/lte/gateway/docker && docker-compose restart")
-        print("Waiting for all services to restart. Sleeping for 60 seconds..")
-        self.wait_for_restart_to_finish(60)
+
+        self._wait_for_pipelined_to_initialize()
```

Comment:
Is `pipelined` the only service we have to wait for? Or the process that takes the longest? 
When we restart the AGW docker containers, `mme` takes the longest amount of time to become healthy. Our (draft) approach is to wait for the containers to report healthy, but we don't have a solid strategy for the `systemd`-base AGW yet. Is the datapath-initialization sufficient? I don't suppose, we could generalize this "health check" for the other services.

Reply:
In the failing tests that this fixes, `pipelined` was the problem. It takes very long, but what's more important is that the first tests after the restart tried to talk to pipelined and failed. This usually happens only in the integtests for the artifacts built with Bazel.

Let's see if it's required for other services as well. I think the wait-for-condition is generally a better pattern than the static sleep duration.